cartography/models/slack/channels.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class SlackChannelNodeProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef('id')
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+    name: PropertyRef = PropertyRef('name', extra_index=True)
+    is_private: PropertyRef = PropertyRef('is_private')
+    created: PropertyRef = PropertyRef('created')
+    is_archived: PropertyRef = PropertyRef('is_archived')
+    is_general: PropertyRef = PropertyRef('is_general')
+    is_shared: PropertyRef = PropertyRef('is_shared')
+    is_org_shared: PropertyRef = PropertyRef('is_org_shared')
+    topic: PropertyRef = PropertyRef('topic.value')
+    purpose: PropertyRef = PropertyRef('purpose.value')
+    num_members: PropertyRef = PropertyRef('num_members')
+
+
+@dataclass(frozen=True)
+class SlackChannelToSlackUserRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+# (:SlackUser)-[:CREATED]->(:SlackChannel)
+class SlackChannelToCreatorRel(CartographyRelSchema):
+    target_node_label: str = 'SlackUser'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'id': PropertyRef('creator')},
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "CREATED"
+    properties: SlackChannelToSlackUserRelProperties = SlackChannelToSlackUserRelProperties()
+
+
+@dataclass(frozen=True)
+# (:SlackUser)-[:MEMBER_OF]->(:SlackChannel)
+class SlackChannelToUserRel(CartographyRelSchema):
+    target_node_label: str = 'SlackUser'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'id': PropertyRef('member_id')},
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "MEMBER_OF"
+    properties: SlackChannelToSlackUserRelProperties = SlackChannelToSlackUserRelProperties()
+
+
+@dataclass(frozen=True)
+class SlackTeamToSlackChannelRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+# (:SlackTeam)<-[:RESOURCE]-(:SlackChannel)
```

Comment:
```suggestion
# (:SlackTeam)-[:RESOURCE]->(:SlackChannel)
```
In AWS, we have the RESOURCE relationship point _from the account to the resource_ like `(:AWSAccount)-[:RESOURCE]->(:EC2Instance)`, so we should follow that pattern here with Slack Team going from the Team to the Channel
https://github.com/lyft/cartography/blob/0c852d26ec32e7dedda28d260290023ea98f3b7e/cartography/models/aws/ec2/instances.py#L53